plugins/extraction/common.ml

```diff
@@ -649,3 +649,64 @@ let get_native_char c =
   Char.chr (cumul l)
 
 let pp_native_char c = str ("'"^Char.escaped (get_native_char c)^"'")
+
+(** Special hack for constants of type String.string : if an
+    [Extract Inductive string => string] has been declared, then
+    the constants are directly turned into string literals *)
+
+let ind_string = mk_ind "Coq.Strings.String" "string"
```

Comment:
I think we should use the `coqlib` machinery here, rather than hardcoding the path. I can take care of it.

Reply:
Done.